[2025-06-08T11:04:20.172+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-08T11:04:20.202+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: large_csv_processor_dag.process_sales_transactions_csv manual__2025-06-08T10:59:05.869653+00:00 [queued]>
[2025-06-08T11:04:20.212+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: large_csv_processor_dag.process_sales_transactions_csv manual__2025-06-08T10:59:05.869653+00:00 [queued]>
[2025-06-08T11:04:20.213+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-08T11:04:20.226+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_sales_transactions_csv> on 2025-06-08 10:59:05.869653+00:00
[2025-06-08T11:04:20.230+0000] {standard_task_runner.py:63} INFO - Started process 811 to run task
[2025-06-08T11:04:20.234+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'large_csv_processor_dag', 'process_sales_transactions_csv', 'manual__2025-06-08T10:59:05.869653+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/csv_processor.py', '--cfg-path', '/tmp/tmpsa0_xrhg']
[2025-06-08T11:04:20.237+0000] {standard_task_runner.py:91} INFO - Job 48: Subtask process_sales_transactions_csv
[2025-06-08T11:04:20.288+0000] {task_command.py:426} INFO - Running <TaskInstance: large_csv_processor_dag.process_sales_transactions_csv manual__2025-06-08T10:59:05.869653+00:00 [running]> on host 6211e2649837
[2025-06-08T11:04:20.379+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='large_csv_processor_dag' AIRFLOW_CTX_TASK_ID='process_sales_transactions_csv' AIRFLOW_CTX_EXECUTION_DATE='2025-06-08T10:59:05.869653+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-08T10:59:05.869653+00:00'
[2025-06-08T11:04:20.381+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-08T11:04:20.394+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-08T11:04:20.395+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/csv_processor.py", line 40, in process_csv_in_chunks
    with open(sql_file_path, 'w') as sql_file:
PermissionError: [Errno 13] Permission denied: './dags/sql/sales_transactions_insert.sql'
[2025-06-08T11:04:20.405+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=large_csv_processor_dag, task_id=process_sales_transactions_csv, run_id=manual__2025-06-08T10:59:05.869653+00:00, execution_date=20250608T105905, start_date=20250608T110420, end_date=20250608T110420
[2025-06-08T11:04:20.417+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 48 for task process_sales_transactions_csv ([Errno 13] Permission denied: './dags/sql/sales_transactions_insert.sql'; 811)
[2025-06-08T11:04:20.455+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-08T11:04:20.481+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.9/site-packages/***/models/baseoperator.py:1296 AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
[2025-06-08T11:04:20.499+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-08T11:04:20.501+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
